pkg/ctl/topic/offload_policies_test.go

```diff
@@ -0,0 +1,71 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package topic
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/streamnative/pulsarctl/pkg/pulsar/utils"
+	"github.com/streamnative/pulsarctl/pkg/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOffloadPoliciesCmd(t *testing.T) {
+	topicName := fmt.Sprintf("persistent://public/default/test-offload-topic-%s", test.RandomSuffix())
+	createArgs := []string{"create", topicName, "1"}
+	_, execErr, _, _ := TestTopicCommands(CreateTopicCmd, createArgs)
+	assert.Nil(t, execErr)
+
+	<-time.After(5 * time.Second)
+	setArgs := []string{"set-offload-policies", topicName,
+		"--driver", "filesystem",
+		"--file-system-uri", "hdfs://127.0.0.1:9000",
+		"--file-system-profile-path", "/pulsarctl/scripts/filesystem_offload_core_site.xml",
```

Comment:
We'd better get the file according to the relative path, so it can be run anywhere. If you are using an absolute path to get the file, it looks like can't be run at local without docker image. You can search `resourceDir()`, I think we are using that in other test.

Reply:
`--file-system-profile-path` is used to tell the pulsar where the file-system-profile-path is, it can be an absolute path, the relative path of pulsar.

Reply:
fixed.